src/textual/widget.py

```diff
@@ -2480,15 +2480,25 @@ def _on_descendant_focus(self, event: events.DescendantBlur) -> None:
         if self._has_focus_within:
             self.app.update_styles(self)
 
-    def _on_mouse_scroll_down(self, event) -> None:
-        if self.allow_vertical_scroll:
-            if self.scroll_down(animate=False):
-                event.stop()
-
-    def _on_mouse_scroll_up(self, event) -> None:
-        if self.allow_vertical_scroll:
-            if self.scroll_up(animate=False):
-                event.stop()
+    def _on_mouse_scroll_down(self, event: events.MouseScrollDown) -> None:
+        if event.ctrl or event.shift:
```

Comment:
Do we want to steal both control and shift here? I forget which one is usually used for this kind of behaviour.

If we steal both, users wouldn't be able to do something like "hold shift and use mouse wheel to zoom instead of scroll", would they?

Reply:
Ideally it would be shift+mouse wheel only, but the problem is that iTerm doesn't report shift when pressed with the mouse wheel. Kitty does. I added both shift and ctrl so there is at least one way of horizontal scrolling.